web/react/components/post_info.jsx

```diff
@@ -99,6 +100,20 @@ export default class PostInfo extends React.Component {
             );
         }
 
+        dropdownContents.push(
+            <li
+                key='copyLink'
+                role='presentation'
+            >
+                <a
+                    href='#'
+                    onClick={(e) => this.setState({target: e.target, show: !this.state.show})}
+                >
+                    {'Permalink'}
```

Comment:
Wondering if we should have a the permanent options at the top of the dropdown list for consistency? 

E.g.: 
- Reply // always there
- Permalink  // always there
- Delete // if user created message or is team admin
- Edit // if user created the message

Right now the first option in the menu changes depending on if Edit and Delete are available and so it's inconsistent. 

Also, in future if we add a keyboard shortcut to open the [...] menu that could be navigated with arrow keys, usage will be more obvious if the order is consistent.